pkg/formatter/table_test.go

```diff
@@ -19,6 +19,7 @@ package formatter_test
 import (
 	"testing"
 
+	"github.com/chef/chef-analyze/pkg/formatter"
```

Comment:
Our test pattern already imports the `formatter` package as `subject` so we can clearly see what
is the subject we are testing. Delete this line and update the other statements to use `subject` instead
of `formatter`

Reply:
Ohh sorry, my bad, thanks for the update